home-tasks/pom.xml

```diff
@@ -12,14 +12,114 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <checkstyle.version>9.2.1</checkstyle.version>
+
+        <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
+        <testng.version>7.5</testng.version>
+        <assertj-core.version>3.22.0</assertj-core.version>
+        <epam-calculator.version>1.0</epam-calculator.version>
+        <slf4j-simple.version>1.7.35</slf4j-simple.version>
+        <testng.simpleSuite.config.path>src/test/resources/suites</testng.simpleSuite.config.path>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>7.4.0</version>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${assertj-core.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.epam.calculator</groupId>
+            <artifactId>epam-calculator</artifactId>
+            <version>${epam-calculator.version}</version>
+        </dependency>
```

Comment:
Which actions were performed to put this artifact into local repository?

Reply:
I used a command for windows console: 
mvn install:install-file -Dfile=C:\Users\artha\Downloads\HW1_Frameworks\calculator-1.0.jar -DgroupId=com.epam.calculator -DartifactId=epam-calculator -Dversion=1.0 -Dpackaging=jar -DgeneratePom=true